src/renderer/views/wallet/SettingsView.tsx

```diff
@@ -82,7 +84,13 @@ export const SettingsView: React.FC = (): JSX.Element => {
     [ethContext.address$]
   )
 
-  const bitcoinLedgerAddress = useObservableState(bitcoinContext.ledgerAddress$, RD.initial)
+  const bitcoinLedgerMainnetAddress = useObservableState(bitcoinContext.ledgerMainnetAddress$, RD.initial)
+  const bitcoinLedgerTestnetAddress = useObservableState(bitcoinContext.ledgerTestnetAddress$, RD.initial)
```

Comment:
seems if we do this `bitcoinContext.ledgerTestnetAddress$: (network: Network) => LiveData<Error, string>`
we could use it just like 
```ts
const [bitcoinLedgerAddress] = useObservableState(() => bitcoinContext.ledgerTestnetAddress$(network), RD.initial)
```